taxi/urls.py

```diff
@@ -13,38 +13,64 @@
     ManufacturerCreateView,
     ManufacturerUpdateView,
     ManufacturerDeleteView,
+    DriverCreateView,
+    DriverDeleteView,
+    DriverUpdateView,
+    RemoveDriverFromCarView,
+    AssignDriverToCarView,
 )
 
 urlpatterns = [
     path("", index, name="index"),
     path(
         "manufacturers/",
         ManufacturerListView.as_view(),
-        name="manufacturer-list",
+        name="manufacturer_list",
     ),
     path(
         "manufacturers/create/",
         ManufacturerCreateView.as_view(),
-        name="manufacturer-create",
+        name="manufacturer_create",
     ),
     path(
         "manufacturers/<int:pk>/update/",
         ManufacturerUpdateView.as_view(),
-        name="manufacturer-update",
+        name="manufacturer_update",
     ),
     path(
         "manufacturers/<int:pk>/delete/",
         ManufacturerDeleteView.as_view(),
-        name="manufacturer-delete",
+        name="manufacturer_delete",
     ),
-    path("cars/", CarListView.as_view(), name="car-list"),
-    path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
-    path("cars/create/", CarCreateView.as_view(), name="car-create"),
-    path("cars/<int:pk>/update/", CarUpdateView.as_view(), name="car-update"),
-    path("cars/<int:pk>/delete/", CarDeleteView.as_view(), name="car-delete"),
-    path("drivers/", DriverListView.as_view(), name="driver-list"),
+    path("cars/", CarListView.as_view(),
+         name="car_list"),
+    path("cars/<int:pk>/", CarDetailView.as_view(),
+         name="car_detail"),
+    path("cars/create/", CarCreateView.as_view(),
+         name="car_create"),
+    path("cars/<int:pk>/update/", CarUpdateView.as_view(),
+         name="car_update"),
+    path("cars/<int:pk>/delete/", CarDeleteView.as_view(),
+         name="car_delete"),
```

Comment:
Code Style issue: Use `-` to split words in URL identification parameter `name`, not the `_`. Please change `car_delete` to `car-delete`.